### ISSUE TITLE
click version 8.1.3 causes TypeError

### ISSUE THREAD

**Issue by sinanugur** [unioslo/tsd-api-client#97]
Hello. If you install the latest version of "click", it causes a type error. 

```
Traceback (most recent call last):
  File "/Users/sium/opt/miniconda3/bin/tacl", line 5, in <module>
    from tsdapiclient.tacl import cli
  File "/Users/sium/opt/miniconda3/lib/python3.9/site-packages/tsdapiclient/tacl.py", line 441, in <module>
    def cli(
  File "/Users/sium/opt/miniconda3/lib/python3.9/site-packages/click/decorators.py", line 308, in decorator
    _param_memo(f, OptionClass(param_decls, **option_attrs))
  File "/Users/sium/opt/miniconda3/lib/python3.9/site-packages/click/core.py", line 2495, in __init__
    super().__init__(param_decls, type=type, multiple=multiple, **attrs)
TypeError: __init__() got an unexpected keyword argument 'autocompletion'
```

I solved this issue by downgrading click but perhaps it is good to address this in the future.

**Comment by haatveit** [comment]
Hi, this has been resolved in 2f0d7543ddd14114fd513e4353e5cff629a99d79. I guess we can publish a new release including it.

**Comment by haatveit** [comment]
https://pypi.org/project/tsd-api-client/3.5.1/
